test/commands/rome-defaults.test.ts

```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@oclif/test'
+
+describe('rome-defaults', () => {
```

Comment:
I think this test should test the creation of a proper file or at least execution of the proper command @korczas